lib/ja_serializer/builder/attribute.ex

```diff
@@ -3,8 +3,24 @@ defmodule JaSerializer.Builder.Attribute do
 
   defstruct [:key, :value]
 
-  def build(%{model: model, serializer: serializer, conn: conn}) do
-    Enum.map apply(serializer,:attributes, [model, conn]), &do_build/1
+  def build(%{model: model, serializer: serializer, conn: conn} = context) do
```

Comment:
I think this is probably throwing compiler warnings, I don't think we need any of the map pattern matching here.